### PR TITLE
trigger rebuild of ddocent

### PR DESCRIPTION
**Pull request by bernt-matthias**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/ddocent/version.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
++++ dDocent-2.7.8/dDocent	2020-11-03 11:07:17.153106953 +0100
+@@ -28,8 +28,8 @@
+ 	fi
+ done
+ 
+-SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c1)
+-SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c3)
++SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f1 -d ".")
++SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f2 -d ".")
++echo "SAMV1 $SAMV1 SAMV2 $SAMV2"
```
Comment (mbargull) [inline recipes/ddocent/version.patch:14]: I assume this is just for debugging purposes in conjunction with the `samtools --version` command in the tests and both will be removed before merging this PR, right?

Comment (mbargull) [reply recipes/ddocent/version.patch:14]: (With "this" I mean the `echo`.)

Comment (bernt-matthias) [reply recipes/ddocent/version.patch:14]: Jep. Thanks.

File: recipes/ddocent/version.patch [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# fix samtools version check
+# (as is latest dev verson of ddocent which is unfortunatelly not released yet)
+diff -ruN dDocent-2.7.8/dDocent dDocent-2.7.8/dDocent
+--- dDocent-2.7.8/dDocent	2019-05-02 20:51:03.000000000 +0200
++++ dDocent-2.7.8/dDocent	2020-11-03 11:07:17.153106953 +0100
+@@ -28,8 +28,8 @@
```
Comment (mbargull) [inline recipes/ddocent/version.patch:6]: The patch is malformed. You need to either remove the `echo` line or change this one to
```suggestion
@@ -28,8 +28,9 @@
```
(And of course changing it back when the `echo` line is removed again.)

**Comment by bernt-matthias** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**